model/atmosphere/diffusion/tests/diffusion_tests/test_diffusion.py

```diff
@@ -364,35 +344,9 @@ def test_run_diffusion_single_step(
     )
     prognostic_state = savepoint_diffusion_init.construct_prognostics()
 
-    vertical_config = v_grid.VerticalGridConfig(
-        icon_grid.num_levels,
-        lowest_layer_thickness=lowest_layer_thickness,
-        model_top_height=model_top_height,
-        stretch_factor=stretch_factor,
-        rayleigh_damping_height=damping_height,
-    )
-
-    vertical_params = v_grid.VerticalGrid(
-        config=vertical_config,
-        vct_a=grid_savepoint.vct_a(),
-        vct_b=grid_savepoint.vct_b(),
-        _min_index_flat_horizontal_grad_pressure=grid_savepoint.nflat_gradp(),
-    )
-
     config = construct_diffusion_config(experiment, ndyn_substeps)
-    additional_parameters = diffusion.DiffusionParams(config)
 
     diffusion_granule = diffusion_instance  # the fixture makes sure that the orchestrator cache is cleared properly between pytest runs -if applicable-
```

Comment:
Not to fix in your PR but: 
Can you check with @kotsaloscv whether the above still needed, now that we could construct several instances of the Diffusion for several backends? 

In case it is not ask him to clean it up...

Reply:
We are going to remove the `diffusion_instance` fixture, since Rico resolved the bug with the orchestrator cache. Also, the tests that check multiple steps of the diffusion granule for two different backends could be re-written in a better way.
I will make sure to have a PR for this.